src/main/java/io/github/tryexceptelse/jdex/be/IRolodex.java

```diff
@@ -16,26 +17,29 @@
      */
     ArrayList<Contact> getContacts();
 
-    /** Returns Contact after adding it to list
-     *
-     * @param newEntries HashMap of ContactEntries, with String keys.
-     *                   This is the map of the entries that will be stored
-     *                   in the new Contact object.
-     * @return Contact which was added to the rolodex. This is so that the UI
-     * has a reference to the newly added object, from which it may display
-     * information to the user, if needed.
+    /**
+     * @param first:      expects FirstName object representing the first name of the contact
+     * @param last:       expect LastName object representing the last name of the contact
+     * @param email:      expects EmailAddress object representing the email address of the contact
+     * @param streetAddr: expects EmailAddress object represnting the email address of the contact
+     * @param phone:      expects PhoneNumber object representing the phone number of the contact
+     * @param notes:      expects ContactNotes object representing notes for the contact
      */
-    Contact addContact(HashMap<String, ContactEntry> newEntries);
+
+    public Contact addContact(FirstName first,
+                              LastName last,
```

Comment:
Constructor methods of Rolodexshould initialize the contacts ArrayList, or the other methods will not have anything to operate on